cmake/ONEMKL.cmake

```diff
@@ -0,0 +1,7 @@
+find_package(ONEMKL)
+if(NOT ONEMKL_FOUND)
+  message(FATAL_ERROR "Can NOT find ONEMKL cmake helpers module!")
+endif()
+
+list(INSERT ONEMKL_LIBRARIES 0 "-Wl,--no-as-needed")
+list(APPEND ONEMKL_LIBRARIES "-Wl,--as-needed")
```

Comment:
TORCH_XPU_OPS_ONEMKL_LIBRARIES. To algin naming style, or better naming you think.

Reply:
Renamed.